src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -131,99 +131,6 @@ public class MyModel { }
             Assert.Equal(2, result.GeneratedSources.Length);
         }
 
-        [Fact(Skip = "Host outputs disabled - https://github.com/dotnet/razor/issues/8400")]
-        public async Task SourceGenerator_RazorFiles_DesignTime()
```

Comment:
Why are we deleting these tests? I get that we're going to switch to reflection, but we still want the underlying functionality, right?

Reply:
This code's been disabled since we renamed the DLLs. It seems like it's going to remain off until the tooling team fixes the in process loading code and that wont' land until very late 17.9. As such we thought just delete the behavior entirely for 17.9 and bring back in 17.10 once we're in same process, the EA DLL saga is behind us and we're ready to move forward with actuallyusing this.

Reply:
The code cannot stay since the reference is going away, so it wouldn't compile (even the test uses the EA DLL to verify the expected host outputs). When we want to bring back the tests, we can dig it from Git history.